### PR TITLE
Feature/featured content component

### LINKED ISSUE #17: Create ContentFeature component
**Note:** this issue is Dependent on #20 

There's a [few](https://thegymnasium.comk) different [places](https://thegymnasium.com/ux-design) where we have a pattern that represents some kind of featured content:

image image 
image

They all contain:
- [ ] a container box with a white background
- [ ] light grey border all around
- [ ] an optional title ("Courses" / "Webinars")
- [ ] an optional main content container (this is likely where `children` will get rendered)
- [ ] an optional button container with a call to action ("Get started")
- [ ] an optional secondary call to action below the container itself ("View more courses")
- [ ] with varies, and is set by the parent component's display rules
- [ ] slightly rounded corners

### PR DESCRIPTION
**Pull request by alihamoody**
Resolves #17 

# Description
Featured Content card

### CODE REVIEW

**Comment by mbifulco** [comment]
🤔 If the card doesn't work as a base, it's likely that it should be updated, right? The card is meant to be the base component for these bits of UI.  What's the problem you're seeing with it in its current implementation?

**Comment by mbifulco** [comment]
The general idea here is to be able to update `Card` one day in the future if we decide we want everything that looks like a card to have a different behavior or appearance

**Comment by alihamoody** [comment]
The `<Card>` is an outer level component that is closer to a page content (It satisfies 3 out of 4 screenshots in #20), while the 4th screenshot can be implemented using the `<FeaturedContent>`.

IMO, to reuse a component, it is more practical to build small components and compose them into bigger ones. Not the other way round. We want to avoid building *container* type components and build complex logic to render and style the children conditionally.
WDYT?

**Comment by mbifulco** [comment]
Hm - I'm still not sure I follow.  At its core, the card is just a white rectangle with a grey border.  There's nothing about that which precludes it from being a container for a page or a smaller bit of content.

If the issue here is the complexity created by adding a right-hand column within the card, then it's possible that the right-hand-column-flavored card should be a component inherited from Card, just like this component would be.

```jsx
<Card>
  {/* two column layout for the old component */}
</Card>
```

is not that different from
```jsx
<Card>
  {/* three row layout for this component */}
</Card>
```

right?

**Comment by alihamoody** [comment]
I see your point. Let me review and get back to you.

File: src/components/FeaturedContent/FeaturedContent.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    color: ${color.grey.dark};
+  `;
+
+  const extraStyle = css`
+    background-color: transparent;
+    padding: 1.5rem 1rem 1rem 1rem;
+    text-align: center;
+  `;
+
+  const zzzmainStyle = css`
```
[the remark this answers is not in the document]
Comment (alihamoody) [reply src/components/FeaturedContent/FeaturedContent.tsx]: @mbifulco I removed the unused reference.
The linter highlighted this but I missed it. 
I didn't configure the lint to block commits. Warnings in the console and highlights in the editor currently show up.

**Comment by alihamoody** [comment]
@mbifulco Could you please review and merge this?

**Pull request merged**